flask_potion/__init__.py

```diff
@@ -182,8 +193,7 @@ def add_route(self, route, resource, endpoint=None, decorator=None):
 
         view = self.output(view_func)
```

Comment:
Sorry I meant to comment on https://github.com/biosustain/potion/pull/110#discussion-diff-97231869R199